dex/src/main/scala/com/wavesplatform/dex/settings/MatcherSettings.scala

```diff
@@ -62,7 +61,8 @@ case class MatcherSettings(
   webSockets: WebSocketSettings,
   addressActor: AddressActor.Settings,
   orderEventsCoordinatorActor: OrderEventsCoordinatorActor.Settings,
-  comparisonTool: ComparisonTool.Settings
+  comparisonTool: ComparisonTool.Settings,
+  cli: Option[CliSettings]
```

Comment:
`Option` here is pointless, because we provide the base settings in our `application.conf`

Reply:
Done